<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignalDialog</class>
 <widget class="QDialog" name="SignalDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Signal</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="labelLabel">
     <property name="text">
      <string>Label</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="labelLineEdit">
     <property name="maxLength">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="transducerLabel">
     <property name="text">
      <string>Transducer type</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="transducerLineEdit"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="physDimLabel">
     <property name="text">
      <string>Physical dimensions</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="physicalDimLineEdit">
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="physMinLabel">
     <property name="text">
      <string>Physical minimum</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="physicalMinLineEdit">
     <property name="text">
      <string>-1</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="physMaxLabel">
     <property name="text">
      <string>Physical maximum</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="physicalMaxLineEdit">
     <property name="text">
      <string>1</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="digMinLabel">
     <property name="text">
      <string>Digital minimum</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="digitalMinLineEdit">
     <property name="text">
      <string>-1</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="digMaxLabel">
     <property name="text">
      <string>Digital maximum</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="digitalMaxLineEdit">
     <property name="text">
      <string>1</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="prefilteringLabel">
     <property name="text">
      <string>Prefiltering</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPlainTextEdit" name="prefilteringTextEdit"/>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SignalDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SignalDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
